Jenkinsfile

```
@@ -1,5 +1,6 @@
-buildPlugin(useContainerAgent: true, configurations: [
-    [ platform: "linux", jdk: "8" ],
-    [ platform: "windows", jdk: "8" ],
-    [ platform: "linux", jdk: "11" ]
+buildPlugin(
+  useContainerAgent: true,
+  configurations: [
+    [platform: 'linux', jdk: 17],
+    [platform: 'windows', jdk: 11],
```

personally I do not like this idiom.  Should change one variable so if something breaks you have a better idea of the cause.
is it java11 or is it windows.
the windows agents are generally less available so you still have to wait around anyway (or they are just broken in which case you have no idea if it even works on java 11)

```suggestion
    [platform: 'linux', jdk: 17],
    [platform: 'linux', jdk: 11],
    [platform: 'windows', jdk: 11],
```